integration-tests/src/test/resources/sitconfig/scripts/runSitConfigTests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -x
+source $HOME/wlserver/server/bin/setWLSEnv.sh
```

Comment:
three comments

- I suspect "ORACLE_HOME" is more portable than "HOME".

- Also, please add a comment to this script to the effect that the test framework copies the script to a WL pod and runs it there...

- Do we really need the '-x' here?  I suspect the 'setWLSEnv.sh' is likely going to generate a lot of tracing that we don't really care about...

Reply:
Fixed to use ORACLE_HOME
Removed -x